packages/plugin-lighthouse/src/lib/runner/utils.ts

```diff
@@ -38,43 +39,46 @@ export class LighthouseAuditParsingError extends Error {
   }
 }
 
+function formatBaseAuditOutput(lhrAudit: Result): AuditOutput {
+  const { id: slug, score, numericValue, displayValue } = lhrAudit;
+  return {
+    slug,
+    score: score ?? 1, // score can be null
+    value: numericValue ?? score ?? 0, // not every audit has a numericValue
+    displayValue:
+      displayValue ??
+      (score === 1 ? 'passed' : score === 0 ? 'failed' : undefined),
+  };
+}
+
+function processAuditDetails(
+  auditOutput: AuditOutput,
+  details: FormattedIcu<Details>,
+): AuditOutput {
+  try {
+    const parsedDetails = toAuditDetails(details);
+    return Object.keys(parsedDetails).length > 0
+      ? { ...auditOutput, details: parsedDetails }
+      : auditOutput;
+  } catch (error) {
+    throw new LighthouseAuditParsingError(auditOutput.slug, error as Error);
+  }
+}
+
 export function toAuditOutputs(
```

Comment:
❓ There is a `lhrToAuditOutputs` function in examples that pretty much mirrors the `toAuditOutputs` logic. Should it also be updated?

Reply:
No. The implementation is the examples is just to showcase. I could use any other less complex CLI there... 
Maybe I can think about a better example that does not overlap with our official plugins or move it into the example-plugins repo.